docs/architecture-cli.md

```diff
@@ -124,7 +128,7 @@ This is accomplished by running the cli tool on any box using the mongo/redis/et
 
 The cli tool will connect directly with the install's db and redis instance(s) so ensure that your box can reach those servers on the appropriate ports.
 
-Also, _please ensure your cli and the server(s) in an environment are using the same version of Talk._ 
+Also, _please ensure your cli and the server(s) in an environment are using the same version of Talk._
```

Comment:
If you have a box that's running cli against an environment with a bunch of talk servers it can be...